test/kubemark/gce/util.sh

```diff
@@ -104,6 +104,21 @@ function delete-master-instance-and-resources {
   fi
 }
 
+function make-kubemark-container {
+  for attempt in $(seq 1 ${RETRIES}); do
+    if ! make gcloudpush; then
```

Comment:
You need to pass the $PROJECT variable to make here, otherwise it will use the default value of `google_containers` for which the person running this script may not have permissions to push. The location where the image is pushed should be same as before.